src/core/metrics/app-metrics.ts

```diff
@@ -127,6 +128,11 @@ export class AppMetrics {
   watchList = METRICS_WATCH_LIST;
 
   watch(name: string) {
+    // TODO: implement watchlists for other modes if necessary
+    // currently only map mode supported
+    const mode = currentModeAtom.getState();
```

Comment:
`.getState()` recalculates atom state and subscribers state (correct me if i'm wrong @Akiyamka ). It used either in debug or when fired once\rarely. Is that the case?

Reply:
it 
> Read (or recalculate) state

https://github.com/artalar/reatom/blob/v2/packages/core/src/types.ts#L93

no subscribing

Reply:
anyway it runs dozens of times (50+)  on app initiating, it's not okay
can you solve this different way?

Reply:
solved via subscribe